Vates/VatesAPI/inc/MantidVatesAPI/PresenterUtilities.h

```diff
@@ -0,0 +1,86 @@
+#ifndef MANTID_VATES_PRESENTER_UTILITIES_H
```

Comment:
Can you maybe split this "utilities" into several headers.
1. Policies.h
2. FactoryChains.h 
....

Wouldn't that be better than having a Utilities dump header? Keep the namespace if you wish.